docs/api.md

```diff
@@ -574,7 +574,8 @@ changing power state, or deprovisioning), but still allows the BMH status
 to be updated unlike the `paused` anotation. While in this state the
 OperationalStatus field will be `detached` but the provisioning state will
 be unmodified.  This API only has any effect for BareMetalHost resources
-that are in either `Provisioned` or `ExternallyProvisioned` state.
+that are in either `Provisioned`, `ExternallyProvisioned`, `Ready` or `Available`
```

Comment:
I usually write this as Ready/Available, since they're the same state. In practice I don't think it's possible to get a Host into the Available state yet.

Reply:
Fixed. thank you